Source/DataSourceProvider.swift

```diff
@@ -37,7 +37,9 @@ where CellFactory.Item == DataSource.Item, SupplementaryFactory.Item == DataSour
     public let supplementaryFactory: SupplementaryFactory
 
     fileprivate var bridgedDataSource: BridgedDataSource?
-
+    
+    //The data source that provides the editing functionality on the tableView
+    fileprivate var dataSourceTableEditing: TableDataSourceEditingController?
```

Comment:
oh, duh! sorry 😄 

Ok, this looks good structurally. Only suggestion then is to follow the style of 
`public var tableViewDataSource: UITableViewDataSource`

let's have a `fileprivate var _tableEditingController` and a `public var tableEditingController` (computed) in the extension 👍